src/appleseed/renderer/kernel/rendering/final/adaptivepixelrenderer.cpp

```diff
@@ -306,6 +306,23 @@ namespace
             return m_params.m_max_samples;
         }
 
+        void print_settings() const override
+        {
+            RENDERER_LOG_INFO(
+                        "adaptive pixel renderer settings:\n"
+                        "  sampling mode                 %s\n"
+                        "  min samples                   %zu\n"
+                        "  max samples                   %zu\n"
+                        "  max variation                 %f\n"
+                        "  diagnostics                   %s",
+                        m_params.m_sampling_mode == SamplingContext::Mode::QMCMode ?
+                            "QMC" : "RMG",
```

Comment:
It's *RNG* for *Random Number Generator*.

Also we have this weird tradition of outputting fully lower case messages so it should be `qmc` and `rng`.